packages/vuetify/src/components/VTreeview/VTreeview.ts

```diff
@@ -157,8 +157,9 @@ export default mixins(
 
   created () {
     this.buildTree(this.items)
-    this.value.forEach(key => this.updateSelected(this.returnObject ? getObjectValueByPath(key, this.itemKey) : key, true))
-    this.active.forEach(key => this.updateActive(this.returnObject ? getObjectValueByPath(key, this.itemKey) : key, true))
+    const getValue = (key: string | number) => this.returnObject ? getObjectValueByPath(key, this.itemKey) : key
```

Comment:
change ternary to 3 lines

Reply:
not sure what you mean here

Reply:
the single line is long (subjective). disregard if you disagree